keyboards/hatsukey70/info.json

```
@@ -0,0 +1,109 @@
+{
+    "manufacturer": "interestor",
+    "keyboard_name": "hatsukey70",
+    "maintainer": "interestor",
+    "bootloader": "atmel-dfu",
```

change to `development_board`
```suggestion
    "development_board": "promicro",
```